schema/deploy/tables/contact.sql

```diff
@@ -42,7 +42,7 @@ comment on column cif.contact.family_name is 'The family name of this contact';
 comment on column cif.contact.email is 'The email address of this contact';
 comment on column cif.contact.phone is 'The phone number of this contact, stored in E.164 format';
 comment on column cif.contact.phone_ext is 'The phone extension of this contact';
-comment on column cif.contact.position is 'The position of this contact';
+comment on column cif.contact.contact_position is 'The position of this contact';
```

Comment:
Position could be the index in the form order, could we specify in this comment that it's the role within a company? Their job title?

```suggestion
comment on column cif.contact.contact_position is 'The role of this contact within their company';
```